src/component/Inputcity.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react'
+import Time from './Time'
+import Weatherrr from './Weatherrr'
+const weather = require('openweather-apis');
+
+function Inputcity() {
+    const [cooord, setcooord] = useState({ lati: '51.5085', long: '-0.1257', cityy: 'London' })
+    const [city, setcity] = useState('London')
```

Comment:
you have a typo here
```suggestion
    const [city, setCity] = useState('London')
```

and maybe you can make it like this

```suggestion
    const [cityInput, setCityInput] = useState('London')
```